### ISSUE TITLE
Avoid node restart upon schema_registry configuration change

### ISSUE THREAD

**Issue by dorlaor** [scylladb/scylladb#15366]
Scylla version (or git commit hash):5.4

Avoid a need to restart upon schema_registry_grace_period change and any other similar configuration
change

**Comment by mykaul** [comment]
@bhalevy - is anyone working on it? I'm deferring it to 6.0, unless we can squeeze it back into 5.4.

**Comment by dani-tweig** [comment]
@dorlaor Is this issue still relevant?

**Comment by dani-tweig** [comment]
@bhalevy , can we move this to the backlog or close it?

**Comment by bhalevy** [comment]
> [@bhalevy](https://github.com/bhalevy) , can we move this to the backlog or close it?

I think so, it shouldn't be need with consistent schema changes over raft, right @tgrabiec @emdroid ?

**Comment by emdroid** [comment]
Yes, I think this shouldn't be needed with raft consistent schema changes.

**Comment by tgrabiec** [comment]
We still have schema_registry with schema-on-raft, and `schema_registry_grace_period` is used to keep non-current table schema versions around.
